data/tlg4170/tlg001b/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4170" xml:lang="grc" urn="urn:cts:greekLit:tlg4170.tlg001b">
+  <ti:title xml:lang="lat">Vita Thomana et hypothesis Olympiorum</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg4170.tlg001b.1st1K-grc1" workUrn="urn:cts:greekLit:tlg4170.tlg001b.1st1K-grc1">
+    <ti:label xml:lang="lat">Vitae Pindari et varia de Pindaro</ti:label>
+    <ti:description xml:lang="mul">Vitae Pindari et varia de Pindaro, Drachmann, Teubner, 1903</ti:description>
```

Comment:
Make the same changes here that I suggested for `data/tlg4170/tlg001a/__cts__.xml` above, obviously with the appropriately changed title.